internal/agentdeployer/_static/docker-agent-base.yml.tmpl

```diff
@@ -18,6 +18,8 @@ services:
       - {{ . }}
     {{- end }}
     {{ end }}
+    cap_drop:
+      - ALL
```

Comment:
What do you think about trying to drop all capabilities also in the main docker compose, and test with integrations?

Reply:
Ok, I'll update those scenarios and run a test with integrations.

Wondering what to do in the template used for custom agents (servicedeployer). For that case, packages could also define some `cap_drop`. It's also true that until now some capabilities are just added with `cap_add`. But looking at the code of `moby`, if ALL is present in cap_drop , all capabilities are drop...

Reply:
It looks like there are no issues when cap_drop ALL is set:
https://github.com/elastic/integrations/pull/9694

Reply:
Should we update also custom agent template ? As there are no packages using `cap_drop` in the integrations repository, probably it's a good idea, WDYT ? 

https://github.com/elastic/elastic-package/blob/5e322d7be61a739b855689aa0c4342abd5fa9d68/internal/servicedeployer/_static/docker-custom-agent-base.yml

Reply:
Yes, good idea.